examples/vm_scheduling/rule_based_algorithm/README.md

```diff
@@ -0,0 +1,29 @@
+# Rule Based Algorithm
+
+## Overview
+
+The VM scheduling problem is one of the quintessential use cases of MARO. The rule based algorithm can be run with a set of scenario configurations that can be found under maro/simulator/scenarios/vm_scheduling. General experimental parameters (e.g., type of topology, type of algorithm to use) can be configured through `config.yml`.
```

Comment:
not of MARO.

the data center counts for the society, while the vm scheduling problem is critical for data center